__tests__/app.test.js

```diff
@@ -167,8 +167,40 @@ describe('GET', () => {
                 });
             });
 
+            test(`200: GET:/api/articles/?topic=mitch get all articles under the mitch topic with all their previously mentioned properties `, () => {
+                return request(app)
+                .get('/api/articles/?topic=mitch')
+                .expect(200)
+                .then(({body: { articles }}) => {
+                    expect(articles.length).toBe(12);
+                    articles.forEach((article) => {
+                        expect(article).not.toHaveProperty('body');
+                        expect(typeof article.author).toBe("string");
+                        expect(typeof article.title).toBe("string");
+                        expect(typeof article.article_id).toBe("number");
+                        expect(article.topic).toBe("mitch");
+                        expect(typeof article.created_at).toBe("string");
+                        expect(typeof article.votes).toBe("number");
+                        expect(typeof article.article_img_url).toBe("string");
+                        expect(typeof article.comment_count).toBe("number");
+                    })
+                });
+            });
```

Comment:
One test missing here:
We have a valid topic with no associated articles (paper)
In this case we would expect 200 responds with an empty array